pkg/tests/ossm/smoke_test.go

```diff
@@ -63,6 +126,36 @@ func TestBookinfoInjection(t *testing.T) {
 	})
 }
 
+func assertProxiesReadyInLessThan3Seconds(t TestHelper, ns string) {
+	t.Log("Extracting proxy startup time and last transition time for all the pods in the namespace")
+	podsList := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.metadata.name}{"\n"}{end}`)
+
+	for _, podName := range strings.Split(podsList, "\n") {
```

Comment:
Because of two things:

1.  In that namespace there is a pod without proxy (sleep pod), skip a pod that does not have proxy time can be bad, because we can skip a fault in the bookinfo installation. So We decided to make the validation in each pod with proxy
2. It's easier to read to anyone when you make the validation in this way

Reply:
@luksa I suggested to simplify it, because it looked more complicated and I had to execute it to understand what it does in fact. It also required more complicated parsing. I didn't know that you suggested to use this command.

Anyway, now the code is simpler to read and understand.